app/controllers/streams_controller.rb

```diff
@@ -1,6 +1,9 @@
 class StreamsController < ApplicationController
-  before_action :set_stream, only: [:show, :edit, :update, :destroy, :playlist, :current_song]
-  before_action :authenticate_admin, :except => [:show, :index, :playlist, :current_song]
+  before_action :set_stream,
+    only: [:show, :edit, :update, :destroy, :playlist, :current_song]
+  before_action :authenticate_admin,
+    :except => [:show, :index, :playlist, :current_song, :subscription]
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.